resources/asciidoctor/lib/care_admonition/extension.rb

```diff
@@ -43,9 +43,25 @@ def initialize(role, default_text)
     end
 
     def process(parent, _target, attrs)
+      text = attrs[:passtext]
```

Comment:
That is super kind of you!

I think it's be cool just to check if it starts with http.  No need for the rest, I think.